src/cloudant/database.py

```diff
@@ -608,6 +608,32 @@ def __getitem__(self, key):
         else:
             raise KeyError(key)
 
+    def __contains__(self, key):
+        """
+        Overrides dictionary __contains__ behavior to check if a document
+        by key exists in the current cached or remote database.
+
+        This allows using checks like
```

Comment:
Replace with `For example:` to be consistent with other code blocks in comments.

Reply:
Done :white_check_mark: